src/components/Branding/branding.css

```diff
@@ -20,6 +20,7 @@
   bottom: -1rem;
   right: 50%;
   transform: translateX(50%);
+  @apply lil-absolute lil-bg-white lil-text-s-18 lil-font-medium lil-px-4 lil-text-font-300;
```

Comment:
tested in all pages ?

Reply:
This is only in enterprise page & tested